roles/lib_openshift/src/doc/volume

```diff
@@ -0,0 +1,98 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_volume
+short_description: Create, modify, and idempotently manage openshift volumes.
+description:
+  - Modify openshift volumes programmatically.
+options:
+  state:
+    description:
+    - State controls the action that will be taken with resource
+    - 'present' will create or update and object to the desired state
+    - 'absent' will ensure volumes are removed
+    - 'list' will read the volumes
+    default: present
+    choices: ["present", "absent", "list"]
+    aliases: []
+  kubeconfig:
+    description:
+    - The path for the kubeconfig file to use for authentication
+    required: false
+    default: /etc/origin/master/admin.kubeconfig
+    aliases: []
+  debug:
```

Comment:
@ashcrow, Its a little difficult to follow but I believe it is here.
https://github.com/kwoodson/openshift-ansible/blob/oc_volume/roles/lib_openshift/src/ansible/oc_volume.py#L14
Then passed into the class here.
https://github.com/kwoodson/openshift-ansible/blob/oc_volume/roles/lib_openshift/src/class/oc_volume.py#L116